console/frontend/src/main/frontend/src/app/views/jms/jms-send-message/jms-send-message.component.ts

```diff
@@ -0,0 +1,126 @@
+import { Component, OnInit } from '@angular/core';
+import { File } from 'src/angularjs/app/app.service';
+import { ApiService } from 'src/angularjs/app/services/api.service';
+
+interface Form {
+    destination: string
+    replyTo: string
+    message: string
+    persistent: boolean
+    propertyValue: string
+    propertyKey: string
+    type: string
+    connectionFactory: string
+    synchronous: boolean
+    lookupDestination: boolean
+    encoding: string
+}
+
+@Component({
+    selector: 'app-jms-send-message',
+    templateUrl: './jms-send-message.component.html',
+    styleUrls: ['./jms-send-message.component.scss']
+})
+export class JmsSendMessageComponent implements OnInit {
+    destinationTypes: string[] = ["QUEUE", "TOPIC"];
+    processing: boolean = false;
+    file: File = {
+        name: ""
+    };
+    connectionFactories: string[] = [];
+    error: string = "";
+    form: Form = {
+        destination: "",
+        replyTo: "",
+        message: "",
+        persistent: false,
+        propertyValue: "",
+        propertyKey: "",
+        type: "",
+        connectionFactory: "",
+        synchronous: false,
+        lookupDestination: false,
+        encoding: ""
+    };
+
+    constructor(
+        private apiService: ApiService
+    ) { };
+
+    ngOnInit(): void {
+        this.apiService.Get("jms", (data) => {
+            this.connectionFactories = data["connectionFactories"];
+            angular.element("select[name='type']").val(this.destinationTypes[0]);
```

Comment:
`angular` is not gonna work when we remove angularjs, maybe try to make this with one-way binding on the actual element(s)